java/ChatSub.java

```diff
@@ -0,0 +1,20 @@
+Channel channel = ably.channels.get("room:tesla-fans");
+
+channel.presence.subscribe(new Presence.PresenceListener() {
```

Comment:
I believe this could be `channel.presence.subscribe(Action.enter, ...)` which would avoid the need for the `if` statement in the handler